docs/pages/blog/pylon-2.0.mdx

```diff
@@ -1,5 +1,7 @@
 ---
 description: 'Pylon v2 introduces the support for different runtimes, a new create command, and an improved development server.'
+date: 'October 10th, 2024'
+new: true
```

Comment:
New is true but I dont see this "New" flag. Why?

![image](https://github.com/user-attachments/assets/207251e3-d189-4dee-869a-1dce6e8de5ca)